controllers/email.js

```diff
@@ -14,33 +14,33 @@
 //NPM DEPENDENCIES
 const nodemailer = require('nodemailer');
 const mailGun = require('nodemailer-mailgun-transport');
+const bcrypt = require('bcrypt');
 
 //cb is callback, cb(err, null) means if err, get err, else null
 //IF WE WANT TO CHANGE THE RECIPIENT ADDRESS LATER, MUST AUTHORIZE IN MAILGUN SYSTEM FIRST
-const sendMail = (email, name, subject, message, cb) => {
-	
+const sendMail = async (from, to, subject,  message) => {
+
 	getTransport()
```

Comment:
Can we write this as async/await instead of a promise?

Reply:
Yes fixed it in the next version